Extend/Payment_Gateway.php

```diff
@@ -9,17 +9,17 @@
 
 namespace Wirecard\Oxid\Extend;
```

Comment:
This class needs to be refactored, too long too many things it does.

Reply:
we can not do this right now, will be done later